mysql-test/suite/rocksdb/t/i_s_ddl.test

```diff
@@ -17,6 +17,7 @@ CREATE TABLE is_ddl_t2 (x INT, y INT, z INT,
                         PRIMARY KEY (z, y) COMMENT 'zy_cf',
                         KEY (x)) ENGINE = ROCKSDB;
 
+--sorted_result
```

Comment:
IIRC I used ORDER BY a while back on some TokuDB issue only to get feedback that --sorted-result might be the more correct mtr semantic, can't remember where, even though I have continued to use ORDER BY to correct non-deterministic ordering for I_S type tables. For some reason, right or wrong, when fixing this, the thought of "should use --sorted-result" came to mind. IMHO it needs to be a rule that some ordering be specified for all non-keyed tables such as I_S and PFS tables as these have caused numerous mtr instabilities over time where results were the same but orders different for whatever 'under the hood' reasons. If you think natural ORDER BY is the way to go, I'm there with you and can easily change this around.